src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
According to the checklist, don't use verbs for class/interface names. It's better to use nouns or noun phrases to represent an interface. Consider renaming this interface to something like `Drawable` which implies that implementing classes will have a drawable behavior.